tests/Unit/Domain/CoordinateMaps/Test_EquatorialCompression.cpp

```diff
@@ -46,36 +47,44 @@ void test_radius() {
   const double aspect_ratio = aspect_ratio_dis(gen);
   CAPTURE(aspect_ratio);
   const std::array<double, 3> input_point{{x, y, z}};
-  const CoordinateMaps::EquatorialCompression angular_compression_map(
-      aspect_ratio);
-  const auto result_point = angular_compression_map(input_point);
-  const auto& result_x = result_point[0];
-  const auto& result_y = result_point[1];
-  const auto& result_z = result_point[2];
 
-  // The EquatorialCompression map should preserve radial lengths.
-  CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+  for (const auto& index_polar_axis : {0_st, 1_st, 2_st}) {
+    const CoordinateMaps::EquatorialCompression angular_compression_map(
+        aspect_ratio, index_polar_axis);
+    const auto result_point = angular_compression_map(input_point);
+
+    // The EquatorialCompression map should preserve radial lengths.
+    CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+
+    // It is easy to mix up \alpha with one over \alpha,
+    // so this comment serves as a reminder to future readers
+    // as to which direction the \alpha should go in the
+    // CHECK statement.
+    //
+    // The aspect ratio is the ratio of the horizontal length
+    // to the vertical height. The parameter name `aspect_ratio` is
```

Comment:
I know you didn't change any wording here, but could you clarify "horizontal length" and "vertical height" here? Maybe using terminology similar to what you used in the doxygen description of `EquitorialCompression.hpp`